analyzers/tests/SonarAnalyzer.UnitTest/TestCases/AsyncVoidMethod.CSharp11.cs

```diff
@@ -6,7 +6,7 @@ public interface ISomeInterface
     {
         static abstract void SomeMethod1();
 
-        static virtual async void SomeVirtualMethod() // FN
+        static virtual async void SomeVirtualMethod() // Compliant (virtual member)
```

Comment:
The rule explicitly ignore abstract/virtual methods as the implementation in the class might end up being non-async.